core/src/components/slides/slides.scss

```diff
@@ -55,3 +55,13 @@
 .swiper-scrollbar-drag {
   background: var(--scroll-bar-background-active);
 }
+
+
+// Pagination
+// Rewrite vendor styles
+// --------------------------------------------------
+.swiper-pagination-fraction,
+.swiper-pagination-custom,
+.swiper-container-horizontal > .swiper-pagination-bullets {
```

Comment:
Why not just `.swiper-pagination-bullets`?

Reply:
I took it from here https://github.com/ionic-team/ionic/blob/master/core/src/components/slides/slides-vendor.scss#L224
I dont sure, but i think vendor's selector, will rewrite `.swiper-pagination-bullets`